src/main/java/org/datadog/jenkins/plugins/datadog/model/GitCommitAction.java

```diff
@@ -140,18 +141,19 @@ public String toString() {
                 '}';
     }
 
-    public static final class ConverterImpl extends DatadogActionConverter {
+    public static final class ConverterImpl extends DatadogActionConverter<GitCommitAction> {
         public ConverterImpl(XStream xs) {
+            super(new ConverterV1());
         }
+    }
 
-        @Override
-        public boolean canConvert(Class type) {
-            return GitCommitAction.class == type;
+    public static final class ConverterV1 extends VersionedConverter<GitCommitAction> {
+        public ConverterV1() {
+            super(1);
```

Comment:
Nit: `super(1)` can be a constant to avoid magic numbers `super(VERSION)`

Reply:
Makes sense, added constants